modules/ROOT/pages/con_pipeline-cluster-resource.adoc

```diff
@@ -0,0 +1,48 @@
+[]
+= Cluster Resource
+
+
+A Cluster Resource represents a Kubernetes cluster other than the current cluster Pipelines is running on. A common use case for this resource is to deploy your application or function on different clusters.
+
+The resource will use the provided parameters to create a
+[kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/)
+file that can be used by other steps in the pipeline task to access the target
+cluster. The kubeconfig will be placed in `/workspace/<your-cluster-name>/kubeconfig` on your Task container
+
+The Cluster resource has the following parameters:
+
+* Required:
+*  `name` (required): The name to be given to the target cluster, will be used
+    in the kubeconfig and also as part of the path to the kubeconfig file
+*   `url` (required): Host url of the master node
+*   `username` (required): the user with access to the cluster
+*   `password`: to be used for clusters with basic auth
+*   `token`: to be used for authentication, if present will be used ahead of the
+    password
+*   `insecure`: to indicate server should be accessed without verifying the TLS
+    certificate.
+*   `cadata` (required): holds PEM-encoded bytes. This is typically read from       a root certificates bundle).
+
+[NOTE]
+====
+Since only one authentication technique is allowed per user, either a `token` or a `password` should be provided, if both are provided, the `password` will be ignored.
```

Comment:
Three points here, @joaedwar.

1. This is what The IBM Style Guide says about the usage of "should: Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as appropriate. 

2. The IBM Style Guide says, "To facilitate translation, use `since` only to refer to time. Do not use it as a synonym for “because.”

3. We could easily make this sentence active: Because Pipelines allows only one authentication technique per user, provide either a `token` or a `password`. If you provide both, Pipelines ignores the `password`.
@joaedwar: Here I assume that the system is "Pipelines". If this is not correct, please use the correct system name instead of "Piplelines".